rollbar-client/test/Rollbar/ClientSpec.hs

```diff
@@ -143,3 +145,28 @@ spec = do
           reportDeploy deploy
 
         deployId `shouldSatisfy` (> 0)
+
+  describe "ToJSON Item" $ do
+    context "when serializing to JSON" $ do
+      let item = Item
+            { itemEnvironment = Environment "test"
+            , itemBody = Body { bodyPayload = PayloadMessage (Message "Test" mempty) }
+            , itemLevel = Just LevelInfo
+            , itemPlatform = Just "haskell"
+            , itemLanguage = Just "Haskell"
+            , itemFramework = Just "GHC"
+            , itemRequest = Nothing
+            , itemServer = Nothing
+            , custom = Just mempty
+            , title = Nothing
+            , uuid = Just "12345"
+            , fingerprint = Nothing
+            , itemNotifier = Notifier "rollbar-client" "1.1.0"
+            }
+          jsonItem = decodeUtf8 $ DBL.toStrict $ encode item
+
+      it "omits fields if they are Nothing values" $
+        T.unpack jsonItem `shouldNotContain` "\"server\""
+
+      it "includes fields if they are Just values" $ 
+        T.unpack jsonItem `shouldContain` "\"platform\":\"haskell\""
```

Comment:
This is missing an empty line at the end of the file. Some external tools rely on files having a linebreak at the end of the file to function properly, so it's best practice to include it (and the GitHub UI warns you like in this case when there is not one). Your editor can be configured to add this on save. For now, do you mind adding it?

<img width="565" alt="Screenshot 2025-03-11 at 11 45 01" src="https://github.com/user-attachments/assets/2edb90db-cd84-42a9-a893-d4c84099fb9e" />